prov/efa/src/efa_av.c

```diff
@@ -385,15 +384,17 @@ int efa_conn_rdm_init(struct efa_av *av, struct efa_conn *conn)
 			return -FI_ENOMEM;
 		}
 
-		err = rxr_raw_addr_to_smr_name(conn->ep_addr, smr_name);
+		err = rxr_raw_addr_to_smr_name(conn->ep_addr, &smr_name);
 		if (err != FI_SUCCESS) {
 			EFA_WARN(FI_LOG_AV,
 				 "rxr_ep_efa_addr_to_str() failed! err=%d\n", err);
```

Comment:
Not related to this CR but should this warning message be `rxr_raw_addr_to_smr_name() failed...`?

Reply:
Yes, will need to fix in separate commit.